TCW/tcw31.xml

```diff
@@ -35,26 +38,114 @@
+            <gi>datatype</gi>) the Stylesheets tests may not fail, but
+            neither will they exercise our new code, that which we
+            most need to test.</p>
+            <p>So we need to tell the Stylesheets to use the new
+            version of the Guidelines (v. 629.515.0α) as the base
+            ODD. The way we do this is just to put a copy of the new
+            version of the Guidelines into the Stylesheets repository.</p>
+          </div>
+          <div>
+            <head>fragility</head>
+            <p>Because the test procedure involves comparing two
+            schemas (the actual output generated by the test, and the
+            output that is expected from the test) using a string
+            comparison tool, even very minor changes to the
+            Guidelines may cause a test to fail. In fact, because the
+            glosses and descriptions of XML constructs are included in
+            the output schemas (so that tools like oXygen can make use
+            of them), there are changes that do not change the actual
+            XML defined by the schema, but nonetheless will cause a
+            test to fail.</p>
+            <p><label>Example 1</label> If the description of the
+            <gi>constraint</gi> element were corrected from <quote>the
+            formal rules of a constraint</quote> to <quote>contains
+            the formal rules of a constraint.</quote>, every test that
+            involves creating a schema that contains the
+            <gi>constraint</gi> element would fail.</p>
+            <p><label>Example 2</label> If the content model of the
+            <gi>foo</gi> element were simplified from
+            <egXML xmlns="http://www.tei-c.org/ns/Examples">
+              <sequence>
+                <elementRef key="bar"/>
+                <elementRef key="bar"/>
+                <elementRef key="bar" minOccurs="0" maxOccurs="unbounded"/>
+              </sequence>
+            </egXML>
+            to just
+            <egXML xmlns="http://www.tei-c.org/ns/Examples">
+              <elementRef key="bar" minOccurs="2" maxOccurs="unbounded"/>
+            </egXML>
+            the same set of documents (those that have two or more
+            <gi>bar</gi> elements as children of each <gi>foo</gi>)
+            would be valid against the new schema. But since the
+            actual grammar is different — it now has <code>bar,
+            bar+</code> where it used to have <code>bar, bar,
+            bar*</code>, the test would fail.</p>
+        </div>
+        <div>
+          <head>solution</head>
+          <p>Since the Stylesheets are tested against a specific
+          version of the Guidelines, it is very helpful to have a
+          (more or less) up-to-date copy of the Guidelines in the
+          Stylesheet repository. If nothing else, this minimizes
+          <q>used the wrong version of P5</q> situations.</p>
```

Comment:
Suggest rewrite for simple clarity: 
“This minimizes the possibility of testing with the wrong version of P5.”

Reply:
✓
Not sure if you intended to replace or retain the “If nothing else,” portions, but I like it so kept it.